### ISSUE TITLE
Don't display "page not found" while loading an experiment or score set.

### ISSUE THREAD

**Issue by jstone-uw** [VariantEffect/mavedb-ui#39]

**Comment by EstelleDa** [comment]
Try to solve this problem. I add a new page and router path for Page Not Found, but it only works when path is invalid. It can't handle invalid scoreset or experiment urns. 

In mavedb-ui item.js page, line 92-95
```
    // TODO catch errors in response
    commit('loadedItem', {item: response.data || null})
} catch (err) {
    console.log(`Error while loading item (URL="${url}")`, err)
}
```
How to get the error or null value is the way I think we can use. I'll keep thinking it next week.

**Comment by jstone-uw** [comment]
I think it's fine to show something like "Page not found" when the URL is well-formed but the score set or experiment doesn't exist. But right now, this message is always displayed briefly until the API request for experiment or score set details succeeds. We could display a "Loading" message during this period, or else display nothing except the toolbar. Since the loading period is brief, the latter might actually be less distracting, I suspect.

**Comment by jstone-uw** [comment]
Released in version 2024.0.0 on 2024-04-15.
